### PR TITLE
Support for more italic style & TypeScript .ts / .tsx

### LINKED ISSUE #4: Italic Variant?
Hi Dan, I love your theme. Would it be possible to release a italic variant for fonts like Operator Mono, Dank Mono etc?

PS: Ah okay you already have that, but unfortunately it seems it is not working properly

### PR DESCRIPTION
**Pull request by tobiastimm**
Fixes #4

### CODE REVIEW

**Comment by palashmon** [comment]
First of all thanks for this PR.
Could you please share some before and after screenshots for `.ts`/`.tsx` file support.

This also closes #19

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules
 *.vsix
 .DS_Store
+test
```
Comment (palashmon) [inline .gitignore]: Why is `test` added here? Was it needed for this PR?

File: package.json [hunk 2]
```diff
@@ -10,30 +10,20 @@
...
   },
   "icon": "icon.png",
-  "keywords": [
-    "theme",
-    "minimal",
-    "oceanic",
-    "dark",
-    "sublime"
-  ],
+  "keywords": ["theme", "minimal", "oceanic", "dark", "sublime"],
```
Comment (palashmon) [inline package.json]: Also, I think `package.json` have been unnecessarily modified here, as code changes here are exactly same. Can you please confirm and revert this change?

File: themes/Subliminal-color-theme.json [hunk 3]
```diff
@@ -246,13 +246,31 @@
...
+        "entity.name.type.ts",
+        "italic",
+        "quote",
+        "type .function",
+        "type.function",
+        "storage.type.class",
+        "keyword.control",
+        "modifier",
+        "this"
+      ],
```
Comment (palashmon) [inline themes/Subliminal-color-theme.json]: I think this change needs to come as a separate PR with before & after screenshots for all of the scopes modified, as we need to check/review first if all the changes look ok or not. Thanks!

File: .gitignore (hunk 1 above)
Comment (tobiastimm) [reply .gitignore]: ah sry, yeah I needed some files for testing and forgot to remove it.

File: package.json (hunk 2 above)
Comment (tobiastimm) [reply package.json]: Yh my prettier setup in viscose changed it. I will revert this change

**Comment by tobiastimm** [comment]
**Before**
subliminal-before-tsx
**After**
subliminal-after-tsx

It is still not 100% optimal, but I'm completely new to vscode theming, maybe some other people have some ideas or improvements.

**Comment by tobiastimm** [comment]
I've removed the italic style on the punctuation. Was kinda annoying for me

**Before**
punctuation-change_before

**After**
punctuation-change_after

**Comment by tobiastimm** [comment]
I will close this PR and create a new one, because I've reworked my fork quite a bit.
